toolkit/imageconfigs/files/imagecustomizer/isoinstaller/attended_config_aarch64_64k.json

```diff
@@ -0,0 +1,169 @@
+{
+ "Disks": null,
+ "SystemConfigs": [
```

Comment:
The `SystemConfigs` array has two entries - one for `full` and one for `core`. The authoring in `full-aarch64-64k.yaml` is specific to full only. So, we might want to remove the extra `core`?